uaclient/cli.py

```diff
@@ -1143,7 +1143,7 @@ def action_collect_logs(args, *, cfg: config.UAConfig):
             "ua status --format json", "{}/ua-status.json".format(output_dir)
         )
         _write_command_output_to_file(
-            "canonical-livepatch status",
+            "/snap/bin/canonical-livepatch status",
```

Comment:
Tiny nit: Should this use the `livepatch.LIVEPATCH_CMD` variable?

Reply:
I thought about that, but I did not want to add a direct entitlement import into the cli module.
But maybe this doesn't matter that much.

What do you think ?

Reply:
@lucasmoura Good point, that is a sign that something is a little weird.

I think its a sign that the general livepatch functionality that isn't directly tied to the entitlement should be refactored into its own module maybe? Not urgent though.

Up to you on whether you think its reasonable or not. I think I still lean towards using the constant, but am going to go ahead and approve

Reply:
Yes, currently we have no way to handle service specifc requests anywhere in the codebase.
I agree that we need a refactoring on that part. I will use the constant and add a comment that should touch this in the near future